build-an-app-with-the-AWS-cloud-development-kit/notes/01-install-aws-cloud-development-kit-cdk-and-create-a-new-project.md

```diff
@@ -2,31 +2,31 @@
 
 **[📹 Video](https://egghead.io/lessons/aws-install-aws-cloud-development-kit-cdk-and-create-a-new-project)**
 
-We'll start by installing the [AWS Cloud Development Kit](https://aws.amazon.com/cdk/).
+00:03 We'll start by installing the [AWS Cloud Development Kit](https://aws.amazon.com/cdk/).
```

Comment:
the timestamps in these markdown files need to be mdx components

```
<TimeStamp start="00:07">

The **markdown**.

</TimeStamp>
```

otherwise we won't be able to easily machine parse them